sdmetrics/utils.py

```diff
@@ -275,3 +275,24 @@ def get_alternate_keys(metadata):
             alternate_keys.append(alternate_key)
 
     return alternate_keys
+
+
+def strip_characters(list_character, column_name):
+    """Strip characters from a column name.
+
+    Args:
+        list_character (list):
+            The list of characters to strip.
+        column_name (string):
+            The column name.
+
+    Returns:
+        string:
+            The column name with the characters stripped.
+    """
+    result = column_name
+    for character in list_character:
+        if character in result:
+            result = result.replace(character, '')
```

Comment:
Does it work if you escape the characters instead of dropping them? As in adding a `\` before the character

Reply:
No I tried doing
`result = result.replace(character, '\\' + character)`
which adds a single `\` before my character but now my test is failing with the initial error:
` ValueError: multi-line expressions are only valid in the context of data, use DataFrame.eval`

Reply:
My only concern is that what if after stripping the characters from the column name, we end up with another column name that is already in the data. It's unlikely but possible right?